README.md

```diff
@@ -1 +1,17 @@
-# sctf-in-s-ane
+sCTF - in/s/ane
+===============
+
+Solutions to problems written by team in/s/ane during the [2015 February sCTF](http://compete.sctf.io) challenge
```

Comment:
This phrase is ambiguous. It sounds like the problems were written by team in/s/ane.